scripts/stop

```diff
@@ -80,7 +80,7 @@ compose() {
       compose up --no-recreate --no-start
 
       # Remove services, images and volumes
-      compose down --remove-orphans --rmi=all --volumes $@
+      compose down --remove-orphans --rmi=local --volumes $@
```

Comment:
`--rmi=local` instructs docker-compose to remove only images built locally, which it identifies by the absence of an `image` property in `docker-compose.yaml`.